src/applications/vaos/tests/new-express-care-request/form-submission.unit.spec.jsx

```diff
@@ -130,6 +131,8 @@ describe('VAOS integration: Express Care form submission', () => {
     ]);
     mockRequestEligibilityCriteria(['983'], requestCriteria);
     mockPreferences('old.email@va.gov');
+    initialState.expressCare.newRequest.facilityId = '983';
```

Comment:
I want to look at refactoring the tests in here now that we have the full flow in place. Ideally we don't mock out redux state info.